<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delete_Employee</class>
 <widget class="QDialog" name="delete_Employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label{
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background:gray;
selection-background-color: darkgray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>242</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton{
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background:skyblue;
selection-background-color: darkgray;
}</string>
   </property>
   <property name="text">
    <string>Load Table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>242</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2{
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background:red;
selection-background-color: darkgray;
}</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>242</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_3{
border: 2px solid gray;
border-radius: 10px;
padding: 0 8px;
background:green;
selection-background-color: darkgray;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Deleting employee's details</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Employee_ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
